apps/data_handler/handlers/loan_states/nostra_alpha/events.py

```diff
@@ -500,26 +502,40 @@ def process_non_interest_bearing_collateral_burn_event(self, event: pd.Series) -
         # Example:
         # https://starkscan.co/event/0x00744177ee88dd3d96dda1784e2dff50f0c989b7fd48755bc42972af2e951dd6_1.
         user = event["data"][0]
-        if user == self.IGNORE_USER:
+        parsed_event = NostraDataParser.parse_non_interest_bearing_collateral_burn_event(event["data"])
+        if user == self.IGNORE_USER | parsed_event == self.IGNORE_USER:
```

Comment:
why did you change this code?
and why parsed event return only user data here if it should return more values?